packages/docs-data/compile-docs-data

```diff
@@ -50,19 +53,26 @@ function generateDocumentalistData() {
     })
         .use(".md", new dm.MarkdownPlugin({ navPage: NAV_PAGE_NAME }))
         .use(
-            /\.d\.ts$/,
+            /\.tsx?$/,
             new dm.TypescriptPlugin({
-                excludeNames: [/Factory$/, /^I.+State$/],
-                excludePaths: ["node_modules/"],
-                includeDefinitionFiles: true,
+                excludeNames: [/I.+State$/],
+                excludePaths: ["node_modules/", "docs-app/"],
             }),
         )
         .use(".scss", new dm.KssPlugin())
-        .documentGlobs("../*/src/**/*", "../*/dist/index.d.ts")
-        .then(docs => JSON.stringify(docs, null, 2))
+        .documentGlobs(`../{${PACKAGES.join(",")}}/src/**/*`)
+        .then(docs => JSON.stringify(docs, transformDocumentalistData, 2))
         .then(content => fs.writeFileSync(path.join(GENERATED_SRC_DIR, DOCS_DATA_FILENAME), content));
 }
 
+function transformDocumentalistData(key, value) {
+    // remove all class declarations as they're currently unused and only increase bundle size
```

Comment:
shouldn't documentalist have an option for this? not a fan of this post-transformation